### PR TITLE
Mattermost Marketing Content Guidelines

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/process/marketing-guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ 6. International. Mattermost has customers around the world and in many cases English is their second language. Use standard American English but do not use metaphors or expressions that people outside of North America are unlikely to understand.
+ 7. Avoiding unintentionally controversy. We might want to stir up a bit of controversy now and again – but only on relevant technological and business topics. Avoid subjects or analogies that might provoke an emotional or hostile response: religion, politics, warfare, personal relationships etc. 
+ 8. Fit for purpose. Is your content designed to persuade (e.g. an opinion piece), promote (e.g. a product description) or instruct (e.g. to guide a user through procedural steps)? Adjust your tone and your language accordingly.
+
+
+Clear? So now let's look at our 12 tips for brilliant writing. To win the reader's attention, and hold it, our content must be a pleasure to read. Our words must sparkle. Yet at the same time, it should all seem effortless; it can be a turnoff if it appears that the reader is trying too hard.
```
Comment (lfbrock) [inline source/process/marketing-guidelines.rst]: should this be "the writer is trying too hard"?

File: source/process/marketing-guidelines.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+ 6. International. Mattermost has customers around the world and in many cases English is their second language. Use standard American English but do not use metaphors or expressions that people outside of North America are unlikely to understand.
+ 7. Avoiding unintentionally controversy. We might want to stir up a bit of controversy now and again – but only on relevant technological and business topics. Avoid subjects or analogies that might provoke an emotional or hostile response: religion, politics, warfare, personal relationships etc. 
+ 8. Fit for purpose. Is your content designed to persuade (e.g. an opinion piece), promote (e.g. a product description) or instruct (e.g. to guide a user through procedural steps)? Adjust your tone and your language accordingly.
+
+
+Clear? So now let's look at our 12 tips for brilliant writing. To win the reader's attention, and hold it, our content must be a pleasure to read. Our words must sparkle. Yet at the same time, it should all seem effortless; it can be a turnoff if it appears that the reader is trying too hard. 
+
+These guidelines help you to achieve the right balance.
```
Comment (lfbrock) [inline source/process/marketing-guidelines.rst]: "help you achieve", also the tense seems weird. Maybe "These guidelines will help you achieve", or "these guidelines are in place to help you achieve"

File: source/process/marketing-guidelines.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+ 7. Avoiding unintentionally controversy. We might want to stir up a bit of controversy now and again – but only on relevant technological and business topics. Avoid subjects or analogies that might provoke an emotional or hostile response: religion, politics, warfare, personal relationships etc. 
+ 8. Fit for purpose. Is your content designed to persuade (e.g. an opinion piece), promote (e.g. a product description) or instruct (e.g. to guide a user through procedural steps)? Adjust your tone and your language accordingly.
+
+
+Clear? So now let's look at our 12 tips for brilliant writing. To win the reader's attention, and hold it, our content must be a pleasure to read. Our words must sparkle. Yet at the same time, it should all seem effortless; it can be a turnoff if it appears that the reader is trying too hard. 
+
+These guidelines help you to achieve the right balance. 
+
+Remember, these are only guidelines, not hard and fast rules that you have to follow slavishly. So, for example, you might describe that in a specific context the passive voice is better than the active (see Point 7).
```
Comment (lfbrock) [inline source/process/marketing-guidelines.rst]: should "describe" be "decide"?

File: source/process/marketing-guidelines.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+ 3. Be specific. Use concrete examples. Don't take things for granted. Explain terms that the reader may be unfamiliar with. For example, do not write "data sovereignty" without explaining the term (e.g. "data sovereignty means the ability to control the country in which your data is stored and the laws under which it may be legally accessed").
+ 4. Choose simple words. Write use rather than utilize or leverage, help rather than facilitate, improve rather than ameliorate, imperfect rather than sub-optimal. It is true that in technical and business writing you sometimes need to choose words that are not in everyday usage, in order to communicate with precision and clarity. An example is the word "compliance". However, if you use long or technical words unnecessarily, inappropriately or out of context you will only cloud your meaning – and look stupid. You would not, for example, say something like "my driving is fully in compliance with current traffic regulations". Instead, you'd use an everyday term like "I observe the rules of the road".
```
Comment (lfbrock) [inline source/process/marketing-guidelines.rst]: do we want to have "and look stupid" in our docs? Not sure if it's professional

File: source/process/marketing-guidelines.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
+============================================================
+Mattermost Marketing Content Guidelines
+============================================================
+
+How to write purposefully and persuasively 
+==============================================================
+
+In this document we set out some easy-to-follow guidelines about writing. Follow them, while also relying on your own judgment. Writing for blogs, newsletters, magazine articles and other forms of content will then become a joy, not a chore.
```
Comment (lfbrock) [inline source/process/marketing-guidelines.rst]: "Writing ... will then become a joy, not a chore" comes off as a little prescriptive / telling people how to feel

**Comment by lfbrock** [comment]
+1 other than comments

File: source/process/marketing-guidelines.rst [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+==============================================================
+
+In this document we set out some easy-to-follow guidelines about writing. Follow them, while also relying on your own judgment. Writing for blogs, newsletters, magazine articles and other forms of content can then become a joy rather than a chore. 
+
+Before we get into the guidelines themselves, let's consider the goal and objectives (why we think written communication is important in the first place) and a few principles (things to bear in mind when pursuing those goals).
+
+Goal
+---------------------------------------------------------
+
+We write marketing content in order to encourage organizations to bring the benefits of modern communication with Mattermost to our target audience.
```
Comment (it33) [inline source/process/marketing-guidelines.rst]: "encourage" seems too weak, propose `Success in marketing means more people in our target audience unlock the benefits of Mattermost for their organizations through faster, larger, more effective deployments.`

File: source/process/marketing-guidelines.rst [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+ 4. Increase the compellingness and trustworthiness of Mattermost product and marketing content as measured by increase in return visits 
+
+The biggest uplift to inbound marketing will come from being featured in online publications such as The Hacker News.
+
+Principles
+---------------------------------------------------------
+
+In order to achieve these goals, our writing must be:
+
+ 1. Magnetic to IT. We need headlines and openings that draw in IT audiences (these include Linux admins, security experts, directors of end-user computing, CIOs). Bear in mind that, although they work in IT, these audiences are typically more interested in the business benefits of technology than technical features.
```
Comment (it33) [inline source/process/marketing-guidelines.rst]: For Principles and Guidelines, propose having first sentence in bold just to have it called out more clearly.

Comment (jasonblais) [inline source/process/marketing-guidelines.rst]: Agree, will update

File: source/process/marketing-guidelines.rst (hunk 6 above)
Comment (it33) [inline source/process/marketing-guidelines.rst]: @jasonblais good point, propose `Success in marketing means more people in our target audience discover, understand, and unlock the benefits of Mattermost for their organizations through faster, larger, more effective deployments.`

^ so discovery leads to understanding leads to unlocking (via deployment)

**Comment by jasonblais** [comment]
Thanks @it33! Updates made

**Pull request merged**